src/main/kotlin/no/nav/syfo/dialogmote/api/v2/DialogmoteActionsApiV2.kt

```diff
@@ -0,0 +1,67 @@
+package no.nav.syfo.dialogmote.api.v2
+
+import io.ktor.application.*
+import io.ktor.http.*
+import io.ktor.request.*
+import io.ktor.response.*
+import io.ktor.routing.*
+import no.nav.syfo.application.api.authentication.getNAVIdentFromToken
+import no.nav.syfo.dialogmote.DialogmoteService
+import no.nav.syfo.dialogmote.api.domain.OvertaDialogmoterDTO
+import no.nav.syfo.dialogmote.tilgang.DialogmoteTilgangService
+import no.nav.syfo.util.callIdArgument
+import no.nav.syfo.util.getBearerHeader
+import no.nav.syfo.util.getCallId
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import java.util.*
+
+private val log: Logger = LoggerFactory.getLogger("no.nav.syfo")
+
+const val dialogmoteActionsApiOvertaPath = "/overta"
+
+fun Route.registerDialogmoteActionsApiV2(
+    dialogmoteService: DialogmoteService,
+    dialogmoteTilgangService: DialogmoteTilgangService,
+) {
+    route(dialogmoteApiV2Basepath) {
+        post(dialogmoteActionsApiOvertaPath) {
```

Comment:
Foreslår å inline path

Reply:
Er det ikke kjekt å ha en const for å kunne gjenbruke i testen?

Reply:
Hm det er sant, hadde ikke sett testen før jeg så denne :D